packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -2686,7 +2686,198 @@ describe('NetworkController', () => {
           );
         });
       });
+      describe('if the controller was destroyed and the all listeners were removed', () => {
+        it('lookupNetwork does not thrown an error', async () => {
```

Comment:
We can understand that test is already testing `lookupNetwork` since it's in a `describe` for one — the "it" here is `lookupNetwork`, in other words — so there is no need to mention it:

``` suggestion
        it('does not throw an error', async () => {